common/test/test_throttle.cpp

```diff
@@ -429,8 +433,8 @@ TEST_P(ThrottlePriorityTest, run) {
 #endif
 
 int main(int argc, char** argv) {
-    if (!photon::is_using_default_engine()) return 0;
```

Comment:
does throttle need to be tested against all event engine?

Reply:
yes